src/controls/RadialProgress.tsx

```diff
@@ -38,8 +45,17 @@ class RadialProgress extends React.Component<IProps, {}> {
   }
 
   public render(): JSX.Element {
-    const { className, data, offset, style, totalRadius } = this.props;
+    const { className, data, offset, style, totalRadius, spin } = this.props;
     const sideLength = (totalRadius + (offset || 0)) * 2;
+
+    if (spin) {
```

Comment:
seems needlessly copy&paste, apart from the classname and data this is the same code as in the default case.

Reply:
I'm also fairly certain this is a bug now. Now the "spin" flag takes precedence over the case where we actually know the progress.